File: Basic_Math/factor_count/factorCount.go
package main

import "fmt"

func factorCount(n int) int {
	var factorCount = 0
	divisors := []int{}
	for i := 1; i*i <= n; i++ {
		if n%i == 0 {
			if n/i == i {
				divisors = append(divisors, i)
				factorCount += 1
			} else {
				divisors = append(divisors, i, n/i)
				factorCount += 2
			}
		}
	}
	fmt.Println(divisors)
	return factorCount
}

func main() {
	fmt.Println("factors counts :", factorCount(36))
}

File: Basic_Math/factor_count/factorCount.py
def factorCount(n):
    factorCount = 0
    divisors = []
    for i in range(1, int(n**0.5) + 1):
        if ( n % i == 0 ):
            if ( n/i == i ):
                divisors.append(i)
                factorCount += 1
            else:
                divisors.append(i)
                divisors.append(int(n/i))
                factorCount += 2
    print(divisors)
    return factorCount

print("factors counts: ",factorCount(36))